src/Map.js

```diff
@@ -78,6 +78,18 @@ export class Map extends KeyedCollection {
     return this.updateIn(keyPath, () => NOT_SET);
   }
 
+  deleteAll(keys) {
+    var iterable = IndexedIterable(keys);
+
+    if (iterable.size === 0) {
+      return this;
+    }
+
+    return this.withMutations(map => {
+      iterable.forEach((key) => map.remove(key, NOT_SET));
```

Comment:
please remove the wrapping `()` for consistency with the previous arrow function argument style